test_files/values.alp

```diff
@@ -15,6 +18,20 @@ let test_atom = :nope
 let test_bool = true
 let test_chars = c"x"
 let test_tuple = (100, "100")
+let test_unit = ()
+
+let wrapped_int = AnInt 41
```

Comment:
@danabr @lepoetemaudit I think maybe we just need to allow for constructors as literal values?  The typer should catch any further issues.